spec/integ/matrix-client-room-timeline.spec.js

```diff
@@ -252,11 +252,14 @@ describe("MatrixClient room timelines", function() {
                 client.scrollback(room).done(function() {
                     expect(room.timeline.length).toEqual(1);
                     expect(room.oldState.paginationToken).toBe(null);
-                    done();
+
+                    // still have a sync to flush
+                    httpBackend.flush("/sync", 1).then(() => {
```

Comment:
We decided (via https://riot.im/develop/#/room/#riot-dev:matrix.org/$1500047765866CodIk:sw1v.org) to leave that discussion for another day.